build.gradle.kts

```diff
@@ -190,6 +192,12 @@ allprojects {
                 roles.add("developer")
               }
             }
+
+            scm {
+              connection.set("scm:git:git@github.com:aws-observability/aws-otel-java-instrumentation.git")
+              developerConnection.set("scm:git:git@github.com:aws-observability/aws-otel-java-instrumentation.git")
+              url.set("git@github.com:aws-observability/aws-otel-java-instrumentation.git")
```

Comment:
This isn't consistent with the URL for the Java SDK: https://github.com/aws/aws-xray-sdk-java/blob/master/build.gradle.kts#L270

Should it be a proper URL or is this format ok too?

Reply:
Good callout - I copied from instrumentation but agree there doesn't seem to be a reason to copy the exact same value all of three times.